test/LightAccountFactory.t.sol

```diff
@@ -59,13 +59,26 @@ contract LightAccountFactoryTest is Test {
         assertEq(address(this).balance, 100 ether);
     }
 
+    function testWithdrawStakeToZeroAddress() public {
+        testUnlockStake();
+        vm.warp(10 hours + 1);
```

Comment:
technically don't need the warp here since the revert happens before the call to EP